using-timescaledb/continuous-aggregates.md

```diff
@@ -0,0 +1,249 @@
+# Continuous Aggregates 
+TimescaleDB continuous aggregates allow users to speed aggregate queries which
+span large amounts of time by continuously calculating them in the background
+and materializing the aggregates in a separate view. Conntinuous aggregates use
+the background worker framework to run aggregates on a schedule as new data is
```

Comment:
presumes knowledge of background worker framework: Do we discuss that somewhere else in the docs (then a link here?) If not, expand a little bit about the background worker framework here.